plasmapy/particles/particle_class.py

```diff
@@ -2244,7 +2254,7 @@ def __hash__(self) -> int:
         Allow use of `hash` so that a |CustomParticle| instance may be used
         as a key in a `dict`.
         """
-        return hash((self.__repr__(), self.symbol))
+        return hash(self.__repr__())
```

Comment:
```suggestion
        return hash(self.symbol)
```
I'm wondering if it would make sense to have the hash be the same as the symbol.  That way, if a `CustomParticle` with a symbol were used as a `dict` key, then it should be possible for the symbol to also be used to access the dict item.  What would your opinion on this be?

```pycon
>>> dilithium = CustomParticle(symbol="dilithium")
>>> d = {dilithium: "..."}
>>> d[dilithium]
>>> d["dilithium"]  # access dict item with string
```

Reply:
That's a really good point.  If we created two different `CustomParticle` instances with the symbol `"I2"`, and put them both in the same`dict`, then using `"I2"` as a key would be ambiguous.  Let's skip this suggestion.  Thanks!